.github/workflows/gradle.yml

```diff
@@ -34,16 +34,3 @@ jobs:
       run: chmod +x gradlew
     - name: Build with Gradle
       run: ./gradlew build
-    # TODO: replace with OpenSearch cluster
-    # TODO: need to bump into 1.13.2 after dealing with the breaking change in ISM: https://github.com/opendistro-for-elasticsearch/index-management/blob/main/release-notes/opendistro-for-elasticsearch-index-management.release-notes-1.13.0.0.md#breaking-changes
```

Comment:
Yes, in this PR, I have OpenSearchSink integration tests separated the above way

- ODFE < 1.13: .github/workflows/opensearch-sink-odfe-before-1_13_0-integration-tests.yml
- ODFE >= 1.13: .github/workflows/opensearch-sink-odfe-since-1_13_0-integration-tests.yml
- OpenSearch: TBD

right now I do not see a need for running the whole DP test suite on different ODFE versions.